service/frontend/wrappers/clusterredirection/api_generated.go

```diff
@@ -116,7 +118,8 @@ func (handler *clusterRedirectionHandler) DescribeTaskList(ctx context.Context,
 		case targetDC == handler.currentClusterName:
 			dp2, err = handler.frontendHandler.DescribeTaskList(ctx, dp1)
 		default:
-			remoteClient, err := handler.GetRemoteFrontendClient(targetDC)
+			var remoteClient frontend.Client
+			remoteClient, err = handler.GetRemoteFrontendClient(targetDC)
 			if err != nil {
 				return err
 			}
```

Comment:
Might be a cleaner solution:
```suggestion
			remoteClient, clientErr := handler.GetRemoteFrontendClient(targetDC)
			if clientErr != nil {
				return clientErr
			}
```